recipes/fluids/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "fluids" %}
+{% set version = "0.1.74" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bd91938b42cb03519b3a98e2e26af9e548cb9060ddd959420b2e9c07d53cc7c4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - numpy >=1.5.0
```

Comment:
You can remove this here and keep it in `run`

Reply:
Sure, thanks